modules/n1ql/pages/n1ql-language-reference/groupby.adoc

```diff
@@ -12,7 +12,7 @@ include::partial$n1ql-language-reference/horizontal-style.adoc[]
 == Purpose
 
 Use the GROUP BY clause to arrange aggregate values into groups of one or more fields.
-This `GROUP BY` clause follows the `WHERE` clause and precedes the optional `LETTING`, `HAVING`, and `ORDER BY` clauses.
+This `GROUP BY` clause follows the `WHERE` clause and precedes the optional `GROUP AS`, `LETTING`, `HAVING`, and `ORDER BY` clauses.
```

Comment:
Only just noticed this. I don't think it's right to say that the GROUP BY CLAUSE _precedes_ GROUP AS, LETTING, HAVING, or ORDER BY, in as much as these are part of the GROUP BY clause, whereas ORDER BY is on the same level. I would rephrase this to say:

```suggestion
This `GROUP BY` clause follows the `WHERE` clause and may contain the optional `GROUP AS`, `LETTING`, and `HAVING` clauses.
```